tasks/cycles/cycles.js

```diff
@@ -6,22 +8,48 @@
  * do..while - doFactorial
  */
 let forFactorial, whileFactorial, doFactorial;
-const n = 10; //10! = 3628800
+let n = 10; //10! = 3628800
+
+forFactorial = 1;
+for (let i = 1; i < n + 1; i++) {
+  forFactorial = forFactorial * i;
+  console.log(i, forFactorial);
```

Comment:
Use console.log() only to test solutions, do not commit it. Please remove console.log()